web_monitoring/db.py

```diff
@@ -45,3 +47,89 @@ def query_import_status(import_id):
                                                 import_id=import_id)
     return requests.get(url,
                         auth=(settings['db_email'], settings['db_password']))
+
+def get_version_uri(version_id, id_type='db', source_type='versionista', get_previous=False):
```

Comment:
Yes, you got that right. Since the purpose of either option is to return the uri, I thought it would make sense to keep both in the same function.
I think splitting it does make it easier to understand but is it the best way when both the functions serve the same purpose.

Reply:
For me, I think the critical consideration is whether the inputs and outputs are the same. In this case, the outputs are compatible, but there are inputs that apply to one case but not another. That's why I would split them up in the case.